dev-util/kdevelop/patches/kdevelop-5.2.1.patchset

```diff
@@ -0,0 +1,25 @@
+From 832e6c610a9319f4dd7bd42b966d754a3cae32a6 Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Zolt=C3=A1n=20Mizsei?= <zmizsei@extrowerk.com>
+Date: Tue, 3 Apr 2018 20:08:02 +0200
+Subject: Haiku supporting patches
+
+
+diff --git a/kdevplatform/util/dbus_socket_transformer/CMakeLists.txt b/kdevplatform/util/dbus_socket_transformer/CMakeLists.txt
+index f9ce6ac..cfebf7b 100644
+--- a/kdevplatform/util/dbus_socket_transformer/CMakeLists.txt
++++ b/kdevplatform/util/dbus_socket_transformer/CMakeLists.txt
+@@ -15,5 +15,10 @@ if(HAVE_MSG_NOSIGNAL)
```

Comment:
If you added this line
```
+# explicit linking to Haiku's separate libnetwork needed due to socket() usage
```
in the patchset, the `@@ -15,5 +15,10 @@` won't be correct. You need to apply the patch without the newly added comment, then add the comment to `kdevplatform/util/dbus_socket_transformer/CMakeLists.txt` and, finally, regenerate the patchset.
Well, you could, instead, edit the `-15,5 +15,10` manually (if you speak `diff` fluently), but the `cfebf7b` in `index f9ce6ac..cfebf7b` would not be _correct_.

Reply:
I knew you will spot it :)